src/sections/TransactionInfoSection.tsx

```diff
@@ -41,10 +41,20 @@ interface ParamTypes {
   id: string
 }
 
+interface TransactionLite {
+  hash: string
+  blockHash: string
+  timestamp: number
+  inputs?: Input[]
+  outputs?: Output[]
+  gasAmount: number
+  gasPrice: string
+}
```

Comment:
Isn't there a similar interface in alephium-js?

Reply:
Yep, very similar, but the differences were causing problems. Feel free to suggest code changes using `TransactionLike`, `Transaction`, `UnconfirmedTransaction`, `ConfirmedTransaction`, but I've tried and it isn't trivial.

Reply:
Isn't this exactly the same? https://github.com/alephium/alephium-js/blob/upgrade-schemas/api/api-explorer.ts#L132-L144

Reply:
Maybe I had a brainfart moment because yep, my visual cortex is interpreting this as exactly the same. :rofl: